backend/src/services/imageService.ts

```
@@ -334,13 +335,23 @@ export class ImageService {
-   * Get paginated list of images in a project
+   * Get paginated list of images in a project with caching
    */
   async getProjectImages(
     projectId: string,
     userId: string,
     options: ImageQueryParams
   ): Promise<PaginatedImages> {
+    const { page, limit, status, sortBy, sortOrder } = options;
+
+    // Create cache key
+    const cacheKey = `images:project:${projectId}:page:${page}:limit:${limit}:status:${status || 'all'}:sort:${sortBy}:${sortOrder}`;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)